package.json

```diff
@@ -9,6 +9,8 @@
   "scripts": {
     "build": "./node_modules/.bin/webpack",
     "start:dev": "webpack serve --mode=development",
+    "dev:https": "webpack serve --mode=development --https --key .cert/localhost-key.pem --cert .cert/localhost.pem",
+    "gen:cert": "mkcert -install && mkdir .cert && cd $_ && mkcert localhost",
```

Comment:
`cd .cert` instead of `cd $_` i think the latter only works in bash

Reply:
sure thing - can add `cross-env` to make super sure it works on Mac & PC